Python/ceval.c

```diff
@@ -5095,12 +5106,14 @@ maybe_dtrace_line(PyFrameObject *frame,
                                        &bounds);
         *instr_lb = bounds.ap_lower;
         *instr_ub = bounds.ap_upper;
+    } else {
+        line = PyFrame_GetLineNumber(frame);
```

Comment:
When dtrace is active and a trace function is unset, ``f->f_lineno`` is set to -1 on all the frames on the stack and this is why we cannot use directly ``f->f_lineno`` in ``maybe_dtrace_line()`` and why the current line number must be obtained from ``PyFrame_GetLineNumber()`` instead. However the call to ``PyFrame_GetLineNumber()`` is only needed when ``PyDTrace_LINE()`` is called.
I will fix that in the next commit.